java/test/src/main/java/test/Ice/adapterDeactivation/Test.ice

```diff
@@ -8,6 +8,7 @@ module Test
 
 interface TestIntf
 {
+    ["java:identifier:_transient"]
     void transient();
```

Comment:
I kept the operation name the same because this is what we call it in every other language's `adapterDeactivation` test.

Reply:
I don't like _transient: leading underscore is for private fields. I suggest `transient_`.

Reply:
Just pick a different name then, such as `transientOp`.

Reply:
Fine, I changed it to `transientOp` in the places where it was absolutely necessary.
Is this our new escaping convention then? We suffix the illegal identifier with it's type?

ie. ` struct while { ... }` is mapped to `whileStruct`?